public/angular/about/about.js

```diff
@@ -0,0 +1,4 @@
+angular.module('fc.about', [])
+.controller('AboutCtrl', function($scope) {
```

Comment:
I do.  Mostly.  I struggle with the tops of angular files, and we do at work too.  So I went with what we did there.  If you indent every multi-line chain with the angular controllers (and directives and such) then you just have lots and lots of indents that aren't really helpful.